src/scheduler/node_info.c

```diff
@@ -3446,6 +3576,15 @@ resources_avail_on_vnode(resource_req *specreq_cons, node_info *node,
 					set_current_aoe(node, resresv->aoename);
 				}
 			}
+
+			is_p = is_powerok(node, resresv, err);
+			if (is_p == NOT_PROVISIONABLE)
+				return 0;
+			else if (is_p == PROVISIONING_NEEDED) {
+				if (resresv->select->total_chunks > 1) {
```

Comment:
This is a hack.  It's from the original provisioning code.  If you look in eval_complex_selspec(), you will see code that will dup the nodes in the case of multiple chunks.  This is really saying, "If the nodes are dup'd, do X, otherwise don't change the node".